docs/task.md

```
@@ -23,13 +25,39 @@ interface GhRef {
+| `terminalTitle` | OSC 観測値 (動的)             | OSC タイトル更新時                     |
+
+#### 表示優先度 (`resolveDisplayTitle` SSOT)
+
+```text
+userTitle 非空 → userTitle
+ELSE ghTitle 非空 → ghTitle
+ELSE terminalTitle 非空 (CLAUDE_PLACEHOLDER "Claude Code" 除外) → terminalTitle
+ELSE undefined (呼び出し側で "New session" or "#N" にフォールバック)
+```
+
+`ghRef` ありの task は最終表示の先頭に `#<number> ` プレフィックスを付ける。
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)